include/CL/sycl/info/device.hpp

```diff
@@ -37,6 +37,7 @@ enum class device_type : unsigned int {
   accelerator,
   custom,
   automatic,
+  defaults,
```

Comment:
Actually `defaults` has been replaced by the `automatic` above already in SYCL 1.2.1.